SCClassLibrary/Common/GUI/PlusGUI/Math/PlotView.sc

```diff
@@ -172,10 +174,10 @@ Plot {
 		^plotBounds.bottom - (val * plotBounds.height); // measures from top left (may be arrays)
 	}
 
-	drawData {
-		var mode = plotter.plotMode;
+	drawData { |mode|
 		var ycoord = this.dataCoordinates;
 		var xcoord = this.domainCoordinates(ycoord.size);
+		mode = (mode ?? plotter.plotMode).asArray;
```

Comment:
this would then just be the instance variable of the `Plot` object.

Reply:
As described above, because there was originally no instance variable for this in `Plot`, in an attempt to keep it global and free the dependency on `Plotter`, I opted to pass this in from outside. I've rethought this though, see more following...